server/data/applicationClient.test.ts

```diff
@@ -64,7 +64,7 @@ describe('ApplicationClient', () => {
       const application = applicationFactory.build()
 
       fakeApprovedPremisesApi
-        .put(paths.applications.update({ id: application.id }))
+        .put(paths.applications.update({ id: application.id }), JSON.stringify({ data: application.data }))
```

Comment:
Why do we have to stringify this and not any of our other client request bodies in the tests?